cmake/modules/tbb.cmake

```diff
@@ -29,6 +29,9 @@ else()
 			BUILD_IN_SOURCE 1
 			BUILD_BYPRODUCTS ${TBB_LIB}
 			INSTALL_COMMAND "")
+		install(FILES "${TBB_LIB}" DESTINATION "${CMAKE_INSTALL_LIBDIR}/${LIBS_PACKAGE_NAME}/")
+		install(DIRECTORY "${TBB_INCLUDE_DIR}/tbb" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${LIBS_PACKAGE_NAME}/")
```

Comment:
Same as `curl` cmake module comment applies. I don't understand whether is it a rule that `DIRECTORY` install shall end with slash or not!